Lombiq.NodeJs.Extensions/package.json

```diff
@@ -50,7 +50,6 @@
     "clean": "run-p --print-label --continue-on-error \"clean:*\"",
     "watch": "run-p --print-label --continue-on-error \"watch:*\"",
     "-": "--------------------------------------------------- ASSETS -------------------------------------------------",
-    "copy:assets": "run-p --print-label --continue-on-error \"build:assets\"",
```

Comment:
You've removed this, it broke backwards compatibility, and [now the tests fail](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3361443242/jobs/5571845184). 

I guess I have to open a Chart.js PR too. 😩

Reply:
I would have done that for you! Since I broke it 😏